### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by anandmit93**
Included REMARKLIN in the list of RPCs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -201,6 +201,7 @@ For RPCs with no information in those columns this information hasn't been deter
 | POSREGVALRD       | Read a position register          |   Y    |      | .10+ |
 | RECPOS            | Teach position (in program)       |   N    |      | .10+ |
 | REGVALRD          | Read a register (int/real)        |   Y    |      | .10+ |
+| REMARKLIN         | Remark/Unremark lines in a TP prog|   N    |      | .30+ |
```
Comment (gavanderhoorn) [inline README.md:204]:
```suggestion
| REMARKLIN         | (Un)remark lines in a TP prog     |   N    |      | .30+ |
```

**Comment by gavanderhoorn** [comment]
If you agree, please merge the suggestion and we can merge the PR.

**Comment by anandmit93** [comment]
I have made a commit to the suggestion.

**Review by gavanderhoorn — APPROVED** [review]

**Comment by gavanderhoorn** [comment]
Thanks :+1:

File: README.md [hunk 2]
```diff
@@ -201,6 +201,7 @@ For RPCs with no information in those columns this information hasn't been deter
 | POSREGVALRD       | Read a position register          |   Y    |      | .10+ |
 | RECPOS            | Teach position (in program)       |   N    |      | .10+ |
 | REGVALRD          | Read a register (int/real)        |   Y    |      | .10+ |
+| REMARKLIN         | (Un)remark lines in a TP prog     |   N    |      | .30+ |
```
[the remark this answers is not in the document]
Comment (anandmit93) [reply README.md:204]: Yes, you are right. It does not work for V9.30 controller. I have tested it for V9.40/43 and it works.

**Pull request merged**